src/commands/support/utf-8.ts

```diff
@@ -25,7 +25,7 @@ const utf8Embed = makeEmbed({
 });
 
 export const utf8: MessageCommandDefinition = {
-    name: 'utf8',
+    name: 'UTF-8',
```

Comment:
This changes the command, I would not make this change. Could add an alias though, but then the documentation also needs to be updated. 
```suggestion
    name: ['utf8', 'UTF-8'],
```

Reply:
I think no one would bother to write UTF-8 the standardized way as a command, so I reverted it.